edx_lint/tamper_evident.py

```diff
@@ -26,21 +26,21 @@ def write(self, text, hashline=u"# {}"):
         be changed to accommodate different file syntaxes.
 
         Arguments:
-            text (unicode string): the contents of the file to write.
+            text (byte string): the contents of the file to write.
 
-            hashline (unicode string): the format of the last line to append to
+            hashline (byte string): the format of the last line to append to
                 the file, with "{}" replaced with the hash.
```

Comment:
In fact, both arguments must be, otherwise the file contents will be encoded with different encodings in different places, which would be unfortunate.